evm/utils/db.py

```diff
@@ -30,3 +41,18 @@ def get_block_header_by_hash(block_hash, db):
     Returns the header for the parent block.
     """
     return db.get_block_header_by_hash(block_hash)
+
+
+def get_empty_root_hash(db):
+    # TODO: Backport and refactor evm.chains.base.BaseChain and
+    # evm.db.chain.BaseChainDB in master branch
+    root_hash = None
+    if db.trie_class is HexaryTrie:
+        root_hash = BLANK_ROOT_HASH
+    elif db.trie_class is BinaryTrie:
+        root_hash = EMPTY_SHA3
+    else:
+        raise ValueError(
```

Comment:
*nitpick*

`NotImplementedError` and instead indicate that `db.trie_class` is not supported.

Maybe handle `db.trie_class is None` separately if that's a posibility.